<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QArtist</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>630</x>
      <y>0</y>
      <width>641</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string>Result</string>
    </property>
    <property name="textFormat" >
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string>Source</string>
    </property>
    <property name="textFormat" >
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="windowTitle" >
     <string>Dicom Viewer</string>
    </property>
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionClose" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuStyle" >
    <property name="title" >
     <string>Style</string>
    </property>
    <addaction name="actionNPR1" />
    <addaction name="separator" />
    <addaction name="actionNPR2_10" />
    <addaction name="actionNPR2_20" />
    <addaction name="actionNPR2_30" />
    <addaction name="separator" />
    <addaction name="actionNPR_3" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionZoom_To_Fit" />
   </widget>
   <widget class="QMenu" name="menuBasic_Operations" >
    <property name="title" >
     <string>Basic Operations</string>
    </property>
    <addaction name="actionGuassian_Blur" />
    <addaction name="actionEdge_Detection" />
    <addaction name="actionCanny" />
    <addaction name="actionSegmentation" />
    <addaction name="actionHalftoning" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuBasic_Operations" />
   <addaction name="menuStyle" />
   <addaction name="menuView" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="actionClose" />
   <addaction name="actionExit" />
  </widget>
  <action name="actionExit" >
   <property name="icon" >
    <iconset>Resource/Shutdown.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F4</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset>Resource/HD.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
   <property name="toolTip" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset>Resource/Folder.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
   <property name="toolTip" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="icon" >
    <iconset>Resource/Blank.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
   <property name="toolTip" >
    <string>New</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset>Resource/Close.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="toolTip" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionGuassian_Blur" >
   <property name="text" >
    <string>Guassian Blur</string>
   </property>
  </action>
  <action name="actionEdge_Detection" >
   <property name="text" >
    <string>Edge Detection</string>
   </property>
  </action>
  <action name="actionNPR1" >
   <property name="text" >
    <string>NPR1</string>
   </property>
  </action>
  <action name="actionNPR2_10" >
   <property name="text" >
    <string>NPR2 TS 10</string>
   </property>
  </action>
  <action name="actionNPR2_20" >
   <property name="text" >
    <string>NPR2 TS 20</string>
   </property>
  </action>
  <action name="actionNPR2_30" >
   <property name="text" >
    <string>NPR2 TS 30</string>
   </property>
  </action>
  <action name="actionZoom_To_Fit" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Zoom To Fit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCanny" >
   <property name="text" >
    <string>Canny</string>
   </property>
  </action>
  <action name="actionSegmentation" >
   <property name="text" >
    <string>Segmentation</string>
   </property>
  </action>
  <action name="actionNPR_3" >
   <property name="text" >
    <string>NPR 3</string>
   </property>
  </action>
  <action name="actionHalftoning" >
   <property name="text" >
    <string>Halftoning</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
